### PR TITLE
Add dice command

### PR DESCRIPTION
**Pull request by kyleplo**
This adds a dice command, as suggested by PeanutButterSandwich on the Koi Discord server. It has four different modes:
`-dice` - roll a 6-sided dice
`-dice <sides>` - roll a dice with a specific amount of sides
`-dice <stuff>` - choose randomly from a list of comma separated stuff
`-dice <min> <max>` - choose randomly between numbers in a range
It also has a help message which will show either if an invalid command is entered or if `-dice help` is entered.

### CODE REVIEW

File: src/fun/dice/DiceCommand.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (args.length === 0) {
+            var rand = Math.floor(Math.random() * 6) + 1;
+            await msg.channel.send("You rolled a " + rand, Constants.black);
+        } else if (args.length === 1 && args[0] === "help"){
+            await msg.channel.send(`Available commands are:
+\`-dice\` - roll a 6-sided dice
+\`-dice <sides>\` - roll a dice with a specific amount of sides
+\`-dice <stuff>\` - choose randomly from a list of comma seperated stuff
+\`-dice <min> <max>\` - choose randomly between numbers in a range
+\`-dice help\` - show this help message`, Constants.black);
```
Comment (azhangvo) [inline src/fun/dice/DiceCommand.ts]: Can you convert this to an embed (and the other messages)? (Also this is registering as an error when I compile it in TS, not sure if there's a library version mismatch right now)

Comment (kyleplo) [reply src/fun/dice/DiceCommand.ts]: Okay, done. I don't know much about TypeScript so I may have broken something.

Comment (kyleplo) [reply src/fun/dice/DiceCommand.ts]: I downloaded typescript and ran the build script, and I think I fixed it.

File: src/fun/dice/DiceCommand.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            var rand = Math.floor(Math.random() * 6) + 1;
+            await msg.channel.send(createBaseEmbed("You rolled a " + rand, Constants.black));
+        } else if (args.length === 1 && args[0] === "help"){
+            await msg.channel.send(createBaseEmbed(`Available commands are:
+\`-dice\` - roll a 6-sided dice
+\`-dice <sides>\` - roll a dice with a specific amount of sides
+\`-dice <stuff>\` - choose randomly from a list of comma separated stuff
+\`-dice <min>-<max>\` - choose randomly between numbers in a range
+\`-dice help\` - show this help message`, Constants.black));
+        } else if (args.length === 1 && !isNaN(parseFloat(args[0]))){
```
Comment (azhangvo) [inline src/fun/dice/DiceCommand.ts]: I'm a bit late on this reply, but the only change left to make here is change this line to
`} else if (args.length === 1 && !isNaN(parseFloat(args[0])) && parseFloat(args[0]).toString() === args[0]) {`
parseFloat is just a bit too good at parsing and even if I enter "1-5" it parses it as 1. This should fix that problem. All the other stuff I've tested and works perfectly, and I'll just match style and formatting when it's fully merged in!

Comment (kyleplo) [reply src/fun/dice/DiceCommand.ts]: Okay, thanks. I think you can just push to my branch like you own it, but I'm not entirely sure.

**Pull request merged**